File: deaf_grandma.js
function deafGrandma() {
  console.log("HEY KID!");
  var goodbye=0;
  prompt.start();
  while(goodbye<2){
    prompt.get(['input'], function(err, result) {
      const input = result.input;
      if(input!=input.toUpperCase()){
        console.log("SPEAK UP, KID!")
      }
      else if(input==""){
        console.log("WHAT?!")
      }
      else if(input=="GOODBYE!"){
        if(goodbye){
          goodbye++;
          console.log("LATER, SKATER!")
        }
        console.log("LEAVING SO SOON?")
        goodbye++;
      }
      else{
        console.log("NO, NOT SINCE 1946!")
      }
    })
  }
}

deafGrandma();

File: deaf_grandma.py
def deaf_grandma():
    print("HEY KID!");
    goodbye=0;
    while(goodbye<2):
        inp = input("")
        if(inp!=inp.upper()):
            print("SPEAK UP, KID!")
        elif(inp==""):
            print("WHAT?!")
        elif(inp=="GOODBYE!"):
            if(goodbye):
                goodbye+=1
                print("LATER, SKATER!")
            print("LEAVING SO SOON?")
            goodbye+=1
        else:
            print("NO, NOT SINCE 1946!")

deaf_grandma()
